### PR TITLE
Fix scene not stable bug

### PR DESCRIPTION
**Pull request by ZappoMan**

### CODE REVIEW

File: libraries/entities/src/EntityTreeElement.cpp [hunk 1]
```diff
@@ -347,6 +348,12 @@ OctreeElement::AppendState EntityTreeElement::appendElementData(OctreePacketData
                     #endif
                     indexesOfEntitiesToInclude << i;
                     numberOfEntities++;
+                } else {
+                    // if the extra data included this entity, and we've decided to not include the entity, then
+                    // we can treat it as if it was completed.
+                    if (entityTreeElementExtraEncodeData->entities.contains(entity->getEntityItemID())) {
+                        entityTreeElementExtraEncodeData->entities.remove(entity->getEntityItemID());
```
Comment (sethalves) [inline libraries/entities/src/EntityTreeElement.cpp]: is the contains line needed?  wont the remove line (re)do the same hash?

**Comment by hifi-gustavo** [comment]
Build console output for commit: 6d211dd4e526130aad3b586724229536b81d21bd available at: https://gist.github.com/a5be6e1474722eba487a<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7283.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7283.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 35d6afbe776227a49ecf36020f792682a841978b available at: https://gist.github.com/a5be6e1474722eba487a<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7283.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7283.dmg

File: libraries/entities/src/EntityTreeElement.cpp (hunk 1 above)
Comment (ZappoMan) [inline libraries/entities/src/EntityTreeElement.cpp]: originally I had some debug code in there too that I only wanted to print when it actually removed the key... I can probably remove the contains() now

**Comment by hifi-gustavo** [comment]
Build console output for commit: 8d4ef28411d5d375ef2a1f9e3838d77fe52b516d available at: https://gist.github.com/a5be6e1474722eba487a<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7283.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7283.dmg

**Pull request merged**